v2/internal/pkg/release/local_stored_collector.go

```diff
@@ -58,29 +51,17 @@ func (o *LocalStorageCollector) ReleaseImageCollector(ctx context.Context) ([]v2
 	o.Log.Debug(collectorPrefix+"setting copy option o.Opts.MultiArch=%s when collecting releases image", o.Opts.MultiArch)
 	var allImages []v2alpha1.CopyImageSchema
 	var imageIndexDir string
-	if o.Opts.IsMirrorToDisk() || o.Opts.IsMirrorToMirror() || o.Opts.IsPrepare() {
+	if o.Opts.IsMirrorToDisk() || o.Opts.IsMirrorToMirror() {
 		releases := o.Cincinnati.GetReleaseReferenceImages(ctx)
```

Comment:
@aguidirh - so if  I understand this correctly even in the enclave we need to stand up a OSUS server to render a cincinnati json payload and also update the registries.conf accordingly

If so I think this might be need to documented (i.e howto) and explained with good detail, this is a big ask IMHO, but hey that's just me

Reply:
During the mirrorToDisk and mirrorToMirror the cincinnati json (graph-data) is saved in the oc-mirror working-dir. So during the diskToMirror the GetReleaseReferenceImages is going to get the related images from the cincinnati json (graph-data), in this way the OSUS server is not required during the mirroring.

Reply:
@aguidirh - thanks for the explanation, I see that the function getGraphData handles that. This is  a grea t improvement !!!